src/wallet/rpc/spend.cpp

```diff
@@ -931,6 +931,26 @@ RPCHelpMan signrawtransactionwithwallet()
     };
 }
 
+// Definition of allowed formats of specifying transaction outputs in
+// `send` and `walletcreatefundedpsbt` RPCs.
+static std::vector<RPCArg> OutputsDoc()
```

Comment:
In ab9c4225332c028a7c84260d5bf634c428698662 "wallet: extract and reuse RPC argument format definition for outputs"

for a followup: This can be used in src/rpc/rawtransaction.cpp as well as the same lines are copied for `createrawtransaction` and `createpsbt`.